service/surf/vendor/elasticsearch/api.py

```diff
@@ -166,7 +159,7 @@ def search(self, search_text: list, drilldown_names=None, filters=None, ordering
         if len(search_text):
             query_string = {
                 "query_string": {
-                    "fields": ["title^2", "title_plain^2", "text", "text_plain", "description", "keywords", "author"],
+                    "fields": ["title^2", "title_plain^2", "text", "text_plain", "description", "keywords", "authors"],
```

Comment:
publishers ontbreekt hier nog?

Reply:
Wel als we daar op willen zoeken ja 😉 . Ik voeg het toe